packages/components/src/components/wallet-card/wallet-card.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import classNames from 'classnames';
+import { localize } from '@deriv/translations';
+import Badge from '../badge';
+import Button from '../button';
+import Icon from '../icon';
+import Text from '../text';
+import { GradientBackground } from '../gradient-background';
+import { WalletIcon } from '../wallet-icon';
+import './wallet-card.scss';
+
+type TWalletCardProps = {
+    balance?: string;
+    currency?: string;
+    icon: JSX.Element;
+    badgeText: string;
+    size?: 'small' | 'medium' | 'large';
+    wallet_name?: string;
+    wallet_state?: 'active' | 'add' | 'added' | 'default' | 'disabled' | 'faded';
+};
+
+const WalletCard: React.FC<React.PropsWithChildren<TWalletCardProps>> = ({
+    balance,
+    currency,
+    icon,
+    badgeText,
+    size,
+    wallet_name,
+    wallet_state = 'default',
+}) => (
+    <div
+        className={classNames('wallet-card', {
+            [`wallet-card--${size}`]: size,
+            [`wallet-card--${wallet_state}`]: wallet_state,
+        })}
+    >
+        {size === 'small' ? (
```

Comment:
@aum-deriv since we add constants for wallet sizes, we can use it inside WalletCard
```suggestion
        {size === wallet_card.sizes.SMALL ? (
```